homeassistant/components/plaato/entity.py

```diff
@@ -69,8 +70,12 @@ def device_info(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes of the monitored installation."""
-        if self._attributes is not None:
-            return self._attributes
+        if self._attributes:
+            return {
+                attr_key: self._attributes[plaato_key]
+                for attr_key, plaato_key in DEVICE_STATE_ATTRIBUTES.items()
+                if plaato_key in self._attributes and self._attributes[plaato_key]
```

Comment:
Can a valid value be `0`? In that case we may want to check against `None` instead?

Reply:
Good point!